plugins/wysiwygarea/plugin.js

```diff
@@ -8,177 +8,177 @@
  *		mode, which handles the main editing area space.
```

Comment:
What happened here? 😄

Reply:
Oh my... just some f...unky format on save... Sorry about that... This file shouldn't be touched.

Reply:
That is some format on save... however, this file was modified just to prove that bender runner generates a nonempty filter for tests. If everything will be OK - those changes should be removed before merge.